### PR TITLE
emailservice

### PR DESCRIPTION
**Pull request by mboosh**

### CODE REVIEW

File: files/emailservice/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "emailservice",
+  "repo": "mboosh/emailservice",
+  "files": {
+    "include": ["*.js", "*.css"]
```
Comment (megawac) [inline files/emailservice/update.json]: You might want to set the basepath to the path to `webapp`

File: files/emailservice/update.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "emailservice",
+  "repo": "mboosh/emailservice",
+  "files": {
+    "basePath": "webapp",
```
Comment (megawac) [inline files/emailservice/update.json]: from a glance I think this should be `src/main/webapp/`

File: files/emailservice/info.ini [hunk 3]
```diff
@@ -0,0 +1,5 @@
+author = "Tim Yelland"
+github = "https://github.com/mboosh/emailservice"
+homepage = "https://github.com/mboosh/emailservice"
+description = ""
+mainfile = "src/main/webapp/js/app.js"
```
Comment (megawac) [inline files/emailservice/info.ini]: I think this is just js/app.js

Comment (mboosh) [inline files/emailservice/info.ini]: Thank you megawac for all your help, appreciate it as a first time user of jdelivr.

**Pull request merged**